src/main/java/orangetaxiteam/cocoman/web/UserController.java

```diff
@@ -47,5 +47,13 @@ public UserDTO updateUser(@PathVariable String id, @RequestBody UserUpdateReques
     public void deleteUser(@PathVariable String id) {
         this.userApplicationService.deleteUser(id);
     }
+
+
+    @GetMapping(value = "/idCheck/{userId}")
```

Comment:
URL에서 users 밑으로 액션 다음 다시 자원의 id가 나오고 있네요.
자원으로 URL을 다 표현할 수 없어서 액션이 URL에 포함될 경우, 액션을 최대한 뒤로 보내는게 좋아보입니다!

```suggestion
    @PostMapping(value = "/{id}/validate")
```